### ISSUE TITLE
Doesn't want to recognize .root file no matter what

### ISSUE THREAD

**Issue by CroLinuxGamer** [skywind3000/asynctasks.vim#30]
I have been trying for a couple of hours to make .root file for my project but asynctasks doesn't wan't to recognize it no matter what I did.

**Comment by skywind3000** [comment]
What did you mean by "doesn't want to recognize" ? any instruction to reproduce it ?

**Comment by CroLinuxGamer** [comment]
I don't have any kind of particular instructions, after using asynctasks for a couple of months I never had a problem with single file projects.
A couple of days before I tried to use it for a bigger project with multiple files and a .root file.
I made .root in the folder of the project and made the needed tasks and opened neovim, I made a simple test file and tried to compile it using asynctacks and I only saw global tasks.
I used CocList tasks, fzf and couple of other intergrations that are showed on the README but wasn't able to see my project tasks.
I also tried to run :AsyncTask _nameoftask_ and still wasn't able to run the task saying there is no [_nameoftask_]
]I seted up in the folder root files in neovim's config. I made double sure of it.

Also sorry for bad English and explanation of my problem I'm not a native english speaker
